### ISSUE TITLE
Atropos trim multi thread stuck in infinite loop on empty files.

### ISSUE THREAD

**Issue by jganbat-fn** [jdidion/atropos#122]
When given 2 empty R1 and R2 fastq files with multi-thread enabled causes it to stuck on waiting forever. 

Atropos version 1.1.28

Here is the bash script to replicate it, 

`test_R1.fastq.gz` and `test_R2.fastq.gz` are empty files. 

```

FQ1=test_R1.fastq.gz
FQ2=test_R2.fastq.gz
TFQ1=${FQ1%%.fastq.gz}.trimmed.fastq.gz
TFQ2=${FQ2%%.fastq.gz}.trimmed.fastq.gz
CORE="-T 4"

atropos trim $CORE \
 	--preserve-order \
 	--no-default-adapters \
 	-a <ANY ADAPTER> \
 	-A <ANY ADAPTER> \
 	-pe1 $FQ1 -pe2 $FQ2 -o $TFQ1 -p $TFQ2

```

**Comment by jdidion** [comment]
Thanks - will get a fix for this out shortly

**Comment by jdidion** [comment]
Fixed in 1.1.29
